Elasticsearch/Search/SearchResponse.php

```diff
@@ -56,4 +59,9 @@ public function getTotalDocuments(): int
     {
         return count($this->hits);
     }
+
+    public function toArray(): array
```

Comment:
only use for old code?

Reply:
Its on the interface, no need to duplicate the doc block